### ISSUE TITLE
Training on unimodal data

### ISSUE THREAD

**Issue by liyi-div** [Jiaxin-Ye/DepMamba#7]
I hope to train your model on the AVEC2014 dataset. 
Currently, I have only extracted the corresponding facial keypoint information and would like to train on facial features first. 
The code I modified is as follows: 
`class DepMamba_SM(BaseNet):

    def __init__(self, audio_input_size=161, video_input_size=136, mm_input_size=256, mm_output_sizes=[256],
                 d_ffn=1024, num_layers=1, dropout=0.1, activation='GELU', causal=False, mamba_config=None):
        super().__init__()

        self.conv_video = nn.Conv1d(video_input_size, mm_input_size, 1, padding=0, dilation=1, bias=False)
        self.enssm_encoder = EnSSM(num_layers,
                                   mm_output_sizes[-1],
                                   [mm_output_sizes[-1]],
                                   d_ffn,
                                   activation=activation,
                                   dropout=dropout,
                                   causal=causal,
                                   mamba_config=mamba_config)

        self.pool = nn.AdaptiveMaxPool1d(1)

        self.output = nn.Linear(mm_output_sizes[-1], 1)
        self.m = nn.Sigmoid()

        #nn.init.xavier_uniform_(self.conv_audio.weight.data)
        nn.init.xavier_uniform_(self.conv_video.weight.data)

    def feature_extractor(self, x, padding_mask=None, a_inference_params=None, v_inference_params=None):
        x = self.conv_video(x.permute(0, 2, 1)).permute(0, 2, 1)
        x = self.enssm_encoder(x)

        if padding_mask is not None:
            x = x * (padding_mask.unsqueeze(-1).float())
            x = x.sum(dim=1) / (padding_mask.unsqueeze(-1).float()
                                ).sum(dim=1, keepdim=False)  # Compute average
        else:
            x = self.pool(x.permute(0, 2, 1)).squeeze(-1)
        return x

    def classifier(self, x):
        return self.output(x)`
I have removed the Multimodal Collaborative SSM module. Could you please confirm if this modification is correct?

**Comment by Lanxioayan** [comment]
Hello, can you tell me the channel to get the AVEC2014 dataset?Thank you very much.
